src/sentry/lint/engine.py

```diff
@@ -110,7 +117,7 @@ def js_lint(file_list=None):
     return has_errors
 
 
-PRETTIER_VERSION = "1.2.2"
+PRETTIER_VERSION = "1.5.3"
```

Comment:
Upgrading will almost certainly re-write a whole ton of JavaScript. You'll need to re-run prettier on JS too (and in getsentry).